libs/types-common/src/Util/Options.hs

```diff
@@ -6,6 +6,7 @@ module Util.Options where
 import Control.Lens
 import Data.Aeson (FromJSON)
 import Data.Aeson.TH
+import Data.Json.Util (toFieldName)
```

Comment:
At some point it will be difficult to see if some of all the Util modules actually come from us or a 3rd party. I wonder if we should create a Wire.Util module or something like that instead?